src/mapping.ts

```diff
@@ -1,6 +1,8 @@
-import { Address, log } from "@graphprotocol/graph-ts";
+import { Address, BigInt, log } from "@graphprotocol/graph-ts";
 import { Transfer } from "../generated/CowProtocolToken/CowProtocolToken";
+import { Holder, Supply } from "../generated/schema";
 
+export const COW_TOKEN = "0xDEf1CA1fb7FBcDC777520aa7f396b4E015F497aB";
```

Comment:
It's confusing to me that in Non_Cirulating_mainnet, there is the vCow token address from Gchain

Reply:
You may have commented on the wrong line here, but if that is the case, then this was a serious typo.

The address I want for vCOW on mainnet non circulating is `0xD057B63f5E69CF1B929b356b579Cba08D7688048`

Reply:
Yes, that is definitely a typo.

Reply:
Fixed!